dashboard/app/controllers/pd/application_dashboard_controller.rb

```diff
@@ -3,18 +3,13 @@ class ApplicationDashboardController < ApplicationController
     before_action :authenticate_user!
 
     def index
-      @permission = nil
-
-      if current_user.permission? UserPermission::WORKSHOP_ADMIN
-        @permission = :workshop_admin
-      end
-
-      unless @permission
-        render_404
-        return
-      end
-
       view_options(full_width: true)
+
+      @script_data = {
+        props: {
+          partnerName: current_user.regional_partners.first.try(:name)
+        }.to_json
+      }
```

Comment:
I couldn't figure out the syntax to do this without the `props: {}` layer, but that seems unnecessary since there is just one item in the hash

Reply:
I don't really see anything wrong with this